src/configuration-requestor.spec.ts

```diff
@@ -14,6 +14,7 @@ import FetchHttpClient, {
   IHttpClient,
   IUniversalFlagConfigResponse,
 } from './http-client';
+import { StoreBackedConfiguration } from './i-configuration';
```

Comment:
I like this i-* file pattern for interfaces. Helps separate out functionally related interfaces, vs putting them all in one large file